### ISSUE TITLE
Multi-character character constant when using the included MAX7219_Font.

### ISSUE THREAD

**Issue by buentead** [JemRF/max7219#2]
If I use the MAX7219_Font included in the max7219.h file, I receive the following error:
```cpp
...\Arduino\libraries\max7219\max7219.h:95:84: warning: multi-character character constant [-Wmultichar]

   {'5',0b1011011},{'6',0b1011111},{'7',0b1110000},{'8',0b1111111},{'9',0b1111011},{'/0',0b0000000},

                                                                                    ^~~~
```

I temporarily fixed it by defining my 'own' font and I replaced '/0' with '\0'. But I'm not sure if it may cause any side effects.

Regards
Adrian

**Comment by JemRF** [comment]
Thanks, I'll fix. You are right it should be \0. Its there to take care of null characters in the text you are displaying on the display.

**Comment by buentead** [comment]
Thank you.

**Comment by JemRF** [comment]
Fixed in V2.0
